src/Sylius/Component/Product/spec/Model/ProductSpec.php

```diff
@@ -288,4 +294,18 @@ function it_is_deleted_only_if_deletion_date_is_in_past()
         $this->setDeletedAt($deletedAt);
         $this->shouldNotBeDeleted();
     }
+
+    function it_is_enabled_by_default()
+    {
+        $this->shouldBeEnabled();
+    }
+
+    function it_is_toggleable()
+    {
+        $this->enable();
+        $this->shouldBeEnabled();
```

Comment:
Since it's enabled by default I would swap the order (disable then enable again).